Assets/Scenes/Menu.unity

```diff
@@ -730,6 +730,22 @@ PrefabInstance:
     serializedVersion: 3
     m_TransformParent: {fileID: 0}
     m_Modifications:
+    - target: {fileID: 3717336014748129540, guid: 0e2ad36662a02c3488301a5286b27635, type: 3}
+      propertyPath: barrels.Array.size
+      value: 3
+      objectReference: {fileID: 0}
+    - target: {fileID: 3717336014748129540, guid: 0e2ad36662a02c3488301a5286b27635, type: 3}
+      propertyPath: barrels.Array.data[3]
+      value: 
+      objectReference: {fileID: 11400000, guid: 51bb200d04523dd469ed0d8fc736ee64, type: 2}
+    - target: {fileID: 3717336014748129540, guid: 0e2ad36662a02c3488301a5286b27635, type: 3}
+      propertyPath: extensions.Array.size
+      value: 3
+      objectReference: {fileID: 0}
+    - target: {fileID: 3717336014748129540, guid: 0e2ad36662a02c3488301a5286b27635, type: 3}
+      propertyPath: extensions.Array.data[2]
+      value: 
+      objectReference: {fileID: 11400000, guid: 75614ad08cd511f4fa856cd443040455, type: 2}
```

Comment:
If this part is what I think it is, you should spply the overrides to the StaticInfo prefab. I'd except no changes in the menu scene for this PR.

Reply:
Please detail

Reply:
![image](https://github.com/hackerspace-ntnu/Prosjekt-Spill-H22/assets/44986729/04e9709a-9696-491b-a6f6-03907cd2f625)
If you add an extension in **1** you need to apply the overrides by following **2** and **3** (yeah the 2 is badly drawn sorry)

Reply:
![image](https://github.com/hackerspace-ntnu/Prosjekt-Spill-H22/assets/44986729/70feea4a-8917-4bb3-b734-54466da4c41b)
I did check if this was the case rn and seems like it

Reply:
i can double check it, sorry for slow pull request review on my end

Reply:
Its good.